README.md

```diff
@@ -146,16 +148,15 @@ On Windows the config file is located at:
 If you run into a problem with Alacritty, please file an issue. If you've got a
 feature request, feel free to ask about it. Keep in mind that Alacritty is very
 much not looking to be a feature-rich terminal emulator with all sorts of bells
-and widgets. It's primarily a cross-platform, blazing fast `tmux` renderer that
-Just Works.
+and widgets.
```

Comment:
Even though this is already incorrect in the current version, I believe the correct phrasing here would be `... with all sorts of bells and whistles.`.

Reply:
heh, I was having a little too much fun eh? Widgets are the bane of many over-complex UIs. Regardless noted, I don't like it anyway.

Reply:
OK, nevermind I think I've fixed it.